doc/setup.md

```diff
@@ -22,9 +22,27 @@
 
 Cable colours are just a suggestion to help differentiate them. You can use whatever convention you like, or get all the same colour.
 
-# Environment Setup
+# Router Setup
 
-This set up is necessary for running the pose solver and GUI. Please see the section that pertains to your operating system below.
+Coming soon.
+
+# Detector Setup
+
+This system is intended to be used over a LAN connecting Raspberry Pi -based detector(s) and a main computer. The Raspberry Pi's run the ```src.main_detector```, while the main computer runs the ```src.main_pose_solver``` and optionally the ```src.gui.gui``` module.
+
+To setup the Raspberry Pi -based detectors, 
+1. Download the latest OS image (.zip file) under Releases on the main web page of this repository. 
+   - Alternatively, run the setup/create_image.sh script on a compatible linux system, which will generate the OS image from scratch
```

Comment:
`create_image.sh` is the development workflow, is it not?

Reply:
`create_image.sh` is the bash script I use to generate the image files for the Raspberry Pi's.